### ISSUE TITLE
fix library path for macOS for demos

### ISSUE THREAD

**Issue by Alexey-T** [Alexey-T/Python-for-Lazarus#26]
>moving to an Intel Mac where
/Library/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7.dylib
exists, and where I use this as the DLL, the sample project
/demos_lazarus/Python_Console
compiles and launches, but never executes any Python code, even a simple print(3+2). Would it be possible to have at least one working Lazarus demo. It is strange that some of the demos in this repository seem to require the a package from a different repository (P4DLaz). I was also unable to get P4Dlaz to install.

**Comment by Alexey-T** [comment]
@neurolabusc to fix for me.

**Comment by neurolabusc** [comment]
Fixed in [legacy fork (2020.07.31)](https://github.com/neurolabusc/Python27-for-Lazarus). Happy to generate a pull request for trunk once the demo programs are operational so the new features can be tested.

**Comment by Alexey-T** [comment]
Now demo_console has OK python path, so it must work?
```

const
  cPyLibraryWindows = 'python37.dll';
  cPyLibraryLinux = 'libpython3.8.so.1.0'; //default in Ubuntu 20.x
  cPyLibraryMac = '/Library/Frameworks/Python.framework/Versions/3.7/lib/libpython3.7.dylib';
  cPyZipWindows = 'python37.zip';
```
